# HG changeset patch
# User Rik <rik@octave.org>
# Date 1388629495 28800
#      Wed Jan 01 18:24:55 2014 -0800
# Branch stable
# Node ID d638db6d045cef65a529ce1a232da22443806805
# Parent  1874a5a142eeeb0e7e8bed37e696cb91ff7218de
doc: Note that dbstop can be used with class methods as well (bug #40958).

* debug.txi: Note that dbstop can be used with class methods as well
(bug #40958).

diff --git a/doc/interpreter/debug.txi b/doc/interpreter/debug.txi
--- a/doc/interpreter/debug.txi
+++ b/doc/interpreter/debug.txi
@@ -76,19 +76,22 @@ result in Octave terminating normally.
 @section Breakpoints
 
 Breakpoints can be set in any Octave function, using the @code{dbstop}
 function.
 
 @DOCSTRING(dbstop)
 
 @noindent
-Note that breakpoints cannot be set in built-in functions
-(e.g., @code{sin}, etc.) or dynamically loaded function (i.e., oct-files).  To
-set a breakpoint immediately on entering a function, the breakpoint
+Breakpoints in class methods are also supported (e.g.,
+@code{dbstop("@@class/method")}).  However, breakpoints cannot be set in
+built-in functions (e.g., @code{sin}, etc.) or dynamically loaded function
+(i.e., oct-files).
+
+To set a breakpoint immediately on entering a function, the breakpoint
 should be set to line 1. The leading comment block will be ignored and
 the breakpoint will be set to the first executable statement in the
 function.  For example:
 
 @example
 @group
 dbstop ("asind", 1)
 @result{} 29
